requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Modified by SignalFx
 # add dependencies in setup.py
-git+https://github.com/rmfitzpatrick/opentracing-python@flask_scope_manager#opentracing
```

Comment:
I believe this is not really required because

1. This repo does not seem to need/use flask scope manager.
2. Flask scope manager actually lives in https://github.com/signalfx/python-flask